gateway/1.5/modules/ROOT/pages/policies-pdk-compile-policies.adoc

```diff
@@ -2,4 +2,36 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:imagesdir: ../assets/images
\ No newline at end of file
+:imagesdir: ../assets/images
+
+If you have completed all the steps in xref:policies-pdk-create-project.adoc[], you can compile the custom policy project.
+
+You must compile the custom policy for your source code and configuration edits to be present in the `target` directories.
+
+After you compile, test your policy with the following tutorials:
+
+* xref:policies-pdk-debug-local.adoc[]
+* xref:policies-pdk-deploy-policies[]
+
+== Before You Begin
+
+Ensure you have completed all the steps in xref:policies-pdk-create-project.adoc[].
+
+== Build Command
+
+To compile the policy, run the `build `command from the policy's root folder:
+
+[source,ssh]
+----
+make build
+----
+
+The `build` command preforms the following actions:
```

Comment:
Are these actually three separate actions?

Does `build-asset-files` not also generate the configuration files mentioned in the last step?

Reply:
The `build` goal internally executes `build-asset-files` goal, because a precondition of compilation is that all asset files are up-to-date.

Reply:
The `build-asset-files` command is exposed as a separate goal because some of the auto-generated files improve the development experience. Eg. when you're navigating the code with an editor like VS Code, the code tooltips will show an up to date version of the policy configuration.
So, in case you want that, you can simply run `build-asset-files` which will not require your code to compile.